File: 0268-missing-number/solution.c
int missingNumber(int* nums, int numsSize) {
    for(int i=0;i<=numsSize;i++)
    {
        int f=0;
        for(int j=0;j<numsSize;j++)
        {
            if(i==nums[j])
            {
                f=1;
            }
        }
        if(f==0)
        {
            return i;
        }
    }
    return 0;
}

File: 0268-missing-number/solution.java
class Solution {
    public int missingNumber(int[] nums) {
        int sum1=0,sum2=0;
        sum2=nums.length*(nums.length+1)/2;
        for(int i=0;i<nums.length;i++){
            sum1+=nums[i];
        }
        return sum2-sum1;
    }
}
